client/main.jsx

```diff
@@ -218,6 +219,210 @@ const ReactContainer = createContainer(() => {
   }
 }, ReactBase);
 
+const VueOther = {
+  name: 'other',
+  render(h) {
+    return h('p', TEXT);
+  },
+  mounted() {
+    logTime();
+  },
+  updated() {
+    logTime();
+  },
+};
+
+const VueTableCell = {
+  name: 'table-cell',
+  props: ['cell'],
+  functional: true,
+  render(h, { props }) {
+    return h('td', [
+      h('button', {
+        on: {
+          click() {
+            console.log('Clicked!');
+          },
+        },
+      }, props.cell.value),
+    ]);
+  },
+};
+
+const VueTableRow = {
+  name: 'table-row',
+  props: ['row'],
+  functional: true,
+  render(h, { props }) {
+    return h('tr',
```

Comment:
You can write render functions directly, or use JSX, or use the vue template compiler. ;)